README.md

```diff
@@ -1,60 +1,120 @@
 # joycontrol
+
+Branch: master->amiibo_edits
```

Comment:
Did you mean to leave this in the readme for the PR?

Reply:
No, that's just there because ppl got confused with what Readme is up to date for what branch.
Just tell me and I'll remove it.